builtin/providers/aws/resource_aws_iot_policy_attachment.go

```diff
@@ -0,0 +1,151 @@
+package aws
+
+import (
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/iot"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsIotPolicyAttachment() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsIotPolicyAttachmentCreate,
+		Read:   resourceAwsIotPolicyAttachmentRead,
+		Update: resourceAwsIotPolicyAttachmentUpdate,
```

Comment:
As this is an attachment, is there *really* an update? Wouldn't ForceNew (destroy attachment and then create a new attachment) be better?